client/my-sites/subscribers/components/subscriber-data-views/subscriber-data-views.tsx

```diff
@@ -72,6 +76,22 @@ const SubscriberDataViews = ( {
 		setSortTerm,
 	} = useSubscribersPage();
 
+	const pageArgs = {
```

Comment:
These are needed for `useUnsubscribeModal`, which we're now accessing directing instead of as a prop.